README.md

```diff
@@ -301,20 +302,17 @@ Make sure your `package.json` includes a `pretest` _script_ (as in
 
 [Jest](https://jestjs.io/)
 
-#### Continuous Integration
+#### Integration tests
 
-[Travis CI](https://travis-ci.org/)
+* [cypress](https://www.cypress.io/)
+* jest & supertest & mongodb-memory-server
 
-`.travis.yml`
+#### Continuous Integration
 
-```yml
-language: node_js
-node_js:
-  - 8
-  - 10
-```
+* [GitHub Actions](https://github.com/features/actions) 
+* [Travis CI](https://travis-ci.org/)
 
-Status badge
+_Note_ product developers have started swapping to Github Actions
```

Comment:
when? we just started the past month

Reply:
I also moved the `Laboratoria/bootcamp` CI/CD from Travis to GitHub Actions :speak_no_evil: